manifests/overlays/dev/secrets/auth/secret-generator.yaml

```diff
@@ -5,3 +5,4 @@ metadata:
 files:
 - secrets/auth/argocd-secret.enc.yaml
 - secrets/auth/dex-server-sa.enc.yaml
+- secrets/auth/srcops.enc.yaml
```

Comment:
hrmm why do we need this for the dev env?

Reply:
No reason right now probably. My reasoning was that we may want to have the same credentials available in both envs, so it behaves the same, no matter where you put the repo... I admit there's no usecase for it now. Do you want me to remove it?

Reply:
no I think that's a fair reason, lgtm